### PR TITLE
Chore/ add liccheck

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Add a package licence compatibility checker matching what we have in open-aea.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [x] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: liccheck.ini [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+fetchai-ledger-api: >=0.0.1
+chardet: >=3.0.4
+certifi: >=2019.11.28
+;TODO: the following are confilctive packages that need to be sorted
+; sub-dep of open-aea-ledger-ethereum-hwi
+hidapi: >=0.13.1
+; shows in pip freesze but not referenced on code
+paramiko: >=3.1.0
+; sub-dep of docker-compose
+websocket-client: >=0.59.0
```
Comment (dvilelaf) [inline liccheck.ini:106]: These need a ticket

Comment (dvilelaf) [reply liccheck.ini:106]: [Opened now](https://trello.com/c/d0j2FsFO/2039-address-license-conflicts-currently-skipped-on-liccheck)

**Comment by dvilelaf** [comment]
I'm cancelling the tests since this PR does not change code.

File: scripts/freeze_dependencies.py
Comment (Adamantios) [inline scripts/freeze_dependencies.py:1]: The whole script is basically a `pip freeze --exclude open-autonomy`

Comment (dvilelaf) [reply scripts/freeze_dependencies.py:1]: Ah, this is some garbage from previous commits. I'll remove.

File: requirements.txt
[the remark this answers is not in the document]
Comment (dvilelaf) [reply requirements.txt:1]: Ah, this is some garbage from previous commits. I'll remove.

**Review by Adamantios — APPROVED** [review]

**Review by angrybayblade — APPROVED** [review]

**Pull request merged**